fsharp-backend/src/LibBackend/BinarySerialization.fs

```diff
@@ -0,0 +1,53 @@
+module LibBackend.BinarySerialization
+
+// Conversion to/from Dark values to binary formats
+
+open System.Threading.Tasks
+open FSharp.Control.Tasks
+
+open Npgsql.FSharp
+open Npgsql
+open LibBackend.Db
+
+open Prelude
+open Tablecloth
+open Prelude.Tablecloth
+
+module PT = LibExecution.ProgramTypes
+
+// TODO inline this file into canvas
```

Comment:
Does this mean "move this to LibBackend/Canvas.fs"?

Reply:
Yes, but it was an error - I copied it from another file by accident.